extensions/wikia/Search/classes/QueryService/Factory.php

```diff
@@ -47,17 +57,18 @@ public function getSolariumClientConfig($forceMaster = false) {
 		$solariumConfig = [
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$solariumConfig". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)